client/lib/analytics/super-props.js

```diff
@@ -16,6 +16,7 @@ const getSuperProps = ( reduxStore ) => ( eventProperties ) => {
 
 	const omitSelectedSite = shouldReportOmitBlogId( eventProperties.path );
 	const selectedSite = omitSelectedSite ? null : getSelectedSite( state );
+	const isDomainOnlySite = selectedSite?.options?.is_domain_only ?? false;
```

Comment:
We have a domainOnlySite selector, which is doing exactly this operation, would be nice to use it, don't you think so?

client/state/selectors/is-domain-only-site.js